sql/2022/12/opportunity-score-wp-sites-lcp-without-fetchpriority-high.sql

```diff
@@ -0,0 +1,63 @@
+# HTTP Archive query to get % of WordPress sites not having fetchpriority='high' on LCP image.
```

Comment:
Can you rename this file to a more concise name? For example `lcp-image-without-fetchpriority-high-opportunity.sql`.

Reply:
Changed